monitor_test.go

```diff
@@ -66,23 +69,35 @@ func TestMonitor(t *testing.T) {
 			if !ok {
 				return
 			}
-			if event.Error != nil {
-				err = fmt.Errorf("monitor err: %s", event.Error)
```

Comment:
the original version of the test seems to be broken, "err" is not checked at all